draft-ietf-jsonpath-base.md

```diff
@@ -1309,96 +1231,124 @@ Queries:
 | `$[?(@ == @)]` | | | Comparison of structured values |
 {: title="Filter selector examples"}
 
-### List Selector
+## Appenders
 
-The list selector allows combining member names, array indices,
-slices, and filters in a single selector.
+A JSONPath query consists of root identifier followed by a sequence of appenders.
+Valid appenders are
 
-Note: The list selector was called "union selector" in
-{{JSONPath-orig}}, as it was intended to solve use cases addressed by
-the union selector in XPath.
-However, the term "union" has the connotation of a set operation that involves
-merging input sets while avoiding duplicates, so the concept was
-renamed into "list selector".
+  * Child appender `[<selectors>]`, where `<selectors>` is one or more of
+    several selector types, which match the nodes to select,
+    separated by commas
+  * Descendant child appender `..[<selectors>]`
+
+The descendant child appender is a child appender that has been made to
+recursively select values within nested objects and arrays by prepending it with `..`.
```

Comment:
It can also select values which are children of the input value, not just those which appear within objects and arrays nested in the input value.